### PR TITLE
Add subscription buttons to nova custom mtl releases

### PR DESCRIPTION
**Pull request by philipandag**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -146,16 +146,20 @@ nav:
...
           - 'Hardware Compatibility List': unified/novacustom/hcl.md
           - 'Firmware transition': unified/novacustom/firmware-transition.md
-          - 'V54 14th Gen':
+          - 'V540TU 14th Gen':
               - 'Releases': variants/novacustom_v540tu/releases.md
               - 'Hardware Configuration Matrix': variants/novacustom_v540tu/hardware-matrix.md
               - 'Test matrix': variants/novacustom_v540tu/test-matrix.md
               - 'Openness score': variants/novacustom_v540tu/openness-score.md
-          - 'V56 14th Gen':
+          - 'V540ND 14th Gen':
```
Comment (mkopec) [inline mkdocs.yml:154]:
```suggestion
          - 'V540TND 14th Gen':
```

Comment (philipandag) [reply mkdocs.yml]: addressed here: [ce52402](https://github.com/Dasharo/docs/pull/860/commits/ce52402c0a72c81d06e346c2c0cee7a18ecbef2c)

Comment (philipandag) [reply mkdocs.yml]: Specific models, like `V540TND` still exist in the `recovery` document. I wanted to join them but there are seperate photos for `TND` and `TNE` models so I decided to skip that for now.

**Review by mkopec — APPROVED** [review]

**Comment by mkopec** [comment]
@philipandag does this PR need a rebase? I see a lot of commits unrelated to the PR title, so I think a rebase might be needed

**Comment by philipandag** [comment]
@mkopec Which branch are you comparing to? Comparing to `master` i see 25 commits.

**Review by mkopec — APPROVED** [review]

**Comment by mkopec** [comment]
Ah because https://github.com/Dasharo/docs/pull/870 was merged to this branch. All good then no problem

**Pull request merged**